package.json

```diff
@@ -238,7 +238,8 @@
     "semi": false,
     "singleQuote": false,
     "trailingComma": "es5",
-    "bracketSpacing": true
+    "bracketSpacing": true,
+    "arrowParens": "avoid"
```

Comment:
Noticed that VSCode updated to 2, so setting this to v1 default until we want to update things codebase-wide